docs/Unicode.md

```diff
@@ -0,0 +1,129 @@
+# Unicode
+
+This document details suggestions on how to handle unicode in Flux as well as how unicode is handled 
+in selected query languages.
+
+## Unicode Summary
+
+UTF-8 is defined to encode code points in 1 to 4 bytes. From Wikipedia, "The first 128 characters 
+(US-ASCII) need one byte. The next 1,920 characters need two bytes to encode, which covers the remainder 
+of almost all Latin-script alphabets, and also Greek, Cyrillic, Coptic, Armenian, Hebrew, Arabic, Syriac, 
+Thaana and N'Ko alphabets, as well as Combining Diacritical Marks. Three bytes are needed for characters 
+in the rest of the Basic Multilingual Plane, which contains virtually all characters in common use 
+including most Chinese, Japanese and Korean characters. Four bytes are needed for characters in the other 
+planes of Unicode, which include less common CJK characters, various historic scripts, mathematical 
+symbols, and emoji (pictographic symbols)."
+
+## Functions and Functionality in Flux affected by unicode
+
+### Length
+
+Should we define length as the number of characters or the number of bytes?
+(Will be the same for the 128 US-ASCII characters that we are used to, but will be different for other
+languages that Flux may want to support.)
+
+### Substring/Slicing/Indexing
+
+Will probably be more convenient to define length as number of values in the resulting array 
+when `strings.split(v: "...", t: "")` is applied. Alternatively, can define length of array 
```

Comment:
I was thinking that length of a string `"influx"` can be defined as `len(strings.split(v: "influx", t: "")`, which would be `6`. And length of a string `"汉字"` will be `2` since `len(strings.split(v: "汉字", t: "")` will be `2`.